### ISSUE TITLE
"SyntaxError: Unexpected end of JSON input" ocuurs when search

### ISSUE THREAD

**Issue by zhibuba** [tldr-pages/tldr-node-client#370]
## Expected behavior
command "tldr -s xxx" can work

## Actual behavior
command "tldr -s xxx" fails due to a javascript error is thrown

## Log, debug output
SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at /home/123/.nvm/versions/node/v16.13.0/pnpm-global/5/node_modules/.pnpm/tldr@3.3.7/node_modules/tldr/lib/search.js:130:21

## Environment
- Operating system - Ubuntu 20.04
- Node.js version - 16.13
- tldr-node-client version - 3.37

**Comment by sarturodev** [comment]
Hi, I can't reproduce this bug, are you using pnpm to install tldr? maybe it triggers the issue.

**Comment by marchersimon** [comment]
@zhibuba does this happen with any search term or just with `xxx`? What about e.g. `command`, or `abcdefgh`?

**Comment by zhibuba** [comment]
This problem has disappeared in my machine... Maybe is due to pnpm's dependcy issues.
